README.md

```diff
@@ -16,6 +16,7 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/spacemeshos/go-spacemesh)](https://goreportcard.com/report/github.com/spacemeshos/go-spacemesh)
 [![Bors enabled](https://bors.tech/images/badge_small.svg)](https://app.bors.tech/repositories/22421)
 <a href="https://godoc.org/github.com/spacemeshos/go-spacemesh"><img src="https://img.shields.io/badge/godoc-LGTM-blue.svg"/></a>
+[![CI](https://github.com/spacemeshos/go-spacemesh/workflows/CI/badge.svg)](https://github.com/spacemeshos/go-spacemesh/actions)
```

Comment:
Not sure we should add a "CI passing" badge (assuming this is what this line does). The way bors works is that the build on `develop` cannot possibly be broken (nothing that breaks `develop` gets merged).

Do you disagree, @lrettig ?

Reply:
You're right, of course, but my thought was that, to the uninitiated observer who doesn't know what bors is, it's nice to see that we're using CI and that our CI build is passing :) It's sort of something I tend to look for when surfing projects and repositories!

Reply:
I initially removed the CI badge and replaced it with the bors badge, hoping that it would be a good replacement (it links to an explanation of what bors is). But having read that a "CI passing" badge is something you look for (this makes sense), I've re-evaluated my position and now I think we should keep it (even though it logically makes no sense).

I wish bors had a badge that statically says "Build passing | guaranteed by bors" or even just "BORS | passing" instead of "BORS | enabled"...

Reply:
Good point. Maybe we can change what the bors badge says 🤔

Reply:
this is the image provided by bors:

https://bors.tech/images/badge_small.svg -> ![bors enabled](https://bors.tech/images/badge_small.svg)

Of course we could create whatever image we want (badges are just images, sometimes the image served is selected based on status, but not in bors' case).